internal/service/advancedclustertpf/common.go

```diff
@@ -0,0 +1,46 @@
+package advancedclustertpf
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/constant"
+	"github.com/spf13/cast"
+	"go.mongodb.org/atlas-sdk/v20241113004/admin"
+)
+
+func FormatMongoDBMajorVersion(version string) string {
+	if strings.Contains(version, ".") {
+		return version
+	}
+	return fmt.Sprintf("%.1f", cast.ToFloat32(version))
+}
+
+func AddIDsToReplicationSpecs(replicationSpecs []admin.ReplicationSpec20240805, zoneToReplicationSpecsIDs map[string][]string) []admin.ReplicationSpec20240805 {
+	for zoneName, availableIDs := range zoneToReplicationSpecsIDs {
+		var indexOfIDToUse = 0
```

Comment:
```suggestion
	 indexOfIDToUse := 0
```

nit: we normally use this convention

Reply:
Thank you. Changed